test/interactions/panZoomInteractionTests.ts

```diff
@@ -1,348 +1,582 @@
-      TestMethods.triggerFakeTouchEvent("touchend", eventTarget, [endPoint], [1] );
-      assert.deepEqual(xScale.domain(), [SVG_WIDTH / 16, SVG_WIDTH * 5 / 16], "xScale inside target component transforms via pinch");
-      assert.deepEqual(xScale2.domain(), initialDomain, "xScale outside of target component does not transform via pinch");
-      svg.remove();
-    });
+        let panZoomInteraction2 = new Plottable.Interactions.PanZoom();
+        panZoomInteraction2.addXScale(xScale2);
+        panZoomInteraction2.attachTo(component2);
 
-    it("Setting the xScales in batch is the same as adding one at a time", () => {
-      let xScale2 = new Plottable.Scales.Linear();
-      panZoomInteraction.addXScale(xScale2);
-      let xScales = panZoomInteraction.xScales();
-      panZoomInteraction.xScales([xScale, xScale2]);
-      assert.deepEqual(xScales, panZoomInteraction.xScales(), "Setting and adding x scales result in the same behavior");
-      svg.remove();
-    });
+        let table = new Plottable.Components.Table([[component, component2]]);
+        table.renderTo(svg);
 
-    it("Setting the yScales in batch is the same as adding one at a time", () => {
-      let yScale2 = new Plottable.Scales.Linear();
-      panZoomInteraction.addYScale(yScale2);
-      let yScales = panZoomInteraction.yScales();
-      panZoomInteraction.yScales([yScale, yScale2]);
-      assert.deepEqual(yScales, panZoomInteraction.yScales(), "Setting and adding y scales result in the same behavior");
-      svg.remove();
-    });
+        let startPoint = { x: SVG_WIDTH / 4, y: SVG_HEIGHT / 2 };
+        let startPoint2 = { x: SVG_WIDTH / 2, y: SVG_HEIGHT / 2 };
+        let endPoint = { x: SVG_WIDTH * 3 / 4, y: SVG_HEIGHT / 2 };
+        let expectedXDomain = domainAfterZoom(startPoint, startPoint2, endPoint, xScale, true);
 
-    it("Adding an already existent xScale does nothing", () => {
-      let oldXScaleNumber = panZoomInteraction.xScales().length;
-      panZoomInteraction.addXScale(panZoomInteraction.xScales()[0]);
-      assert.lengthOf(panZoomInteraction.xScales(), oldXScaleNumber, "Number of x scales is maintained");
-      svg.remove();
-    });
+        TestMethods.triggerFakeTouchEvent( "touchstart", eventTarget, [startPoint, startPoint2], [0, 1] );
+        TestMethods.triggerFakeTouchEvent("touchmove", eventTarget, [endPoint], [1] );
+        TestMethods.triggerFakeTouchEvent("touchend", eventTarget, [endPoint], [1] );
+        assert.deepEqual(xScale.domain(), expectedXDomain, "xScale inside target component transforms via pinch");
+        assert.deepEqual(xScale2.domain(), initialDomain, "xScale outside of target component does not transform via pinch");
+        svg.remove();
+      });
+
+      function domainAfterZoom(startPoint1: Plottable.Point, startPoint2: Plottable.Point, endPoint: Plottable.Point,
+                               scale: Plottable.QuantitativeScale<number>, x: boolean) {
```

Comment:
Should likely specify that this is for pinching.  Also, unsure if we only want 1 `endPoint` parameter.  There's usually only 1 because it's a lot easier to track but this is not always the case.